support/src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/FlowGraphTable.java

```diff
@@ -250,6 +251,10 @@ boolean isEnd() {
             return node instanceof BlockEndNode;
         }
 
+        boolean isExecuted() {
+            return node.getAction(NotExecutedNodeAction.class) == null;
```

Comment:
For my knowledge, what's gonna happen here for old pipeline that didn't have that action? I suppose as they neither had the jelly below, this is gonna appear greyed out right?